### DISCUSSION TITLE
How do I create a new page so that it will not be included in RSS

### DISCUSSION THREAD

**Discussion by BytePrince** [adityatelange/hugo-PaperMod#401] [Q&A]
For example,I want to create a new page called "About",but I don't want to see it in RSS(https://XXX.com/index.xml)
Just like the page https://adityatelange.github.io/hugo-PaperMod/categories/  and https://adityatelange.github.io/hugo-PaperMod/archives/ ,they will not be shown in the RSS.

**Comment by kdkasad** [comment]
You could customize `layouts/_default/rss.xml` and add a line below line 8 with the following:

```go
{{- $pages = (where $pages .Params.excludeFromRSS true) -}}
```

Then in each page that you want to exclude from the RSS feed, just add the following to the front matter:

```yaml
excludeFromRSS: true
```
